platform.md

```diff
@@ -276,70 +276,17 @@ All lifecycle phases:
 - MUST read `CNB_PLATFORM_API` from the execution environment and evaluate compatibility before attempting to parse other inputs (see [Platform API Compatibility](#platform-api-compatibility))
 - MUST give command line inputs precedence over other inputs
 
-#### `detector`
-The platform MUST execute `detector` in the **build environment**
-
-Usage: 
-```
-/cnb/lifecycle/detector \
-  [-app <app>] \
-  [-buildpacks <buildpacks>] \
-  [-group <group>] \
-  [-layers <layers>] \
-  [-log-level <log-level>] \
-  [-order <order>] \
-  [-plan <plan>] \
-  [-platform <platform>]
-```
-
-##### Inputs
-| Input         | Environment Variable    | Default Value             | Description
-|---------------|-------------------------|---------------------------|----------------------
-| `<app>`         | `CNB_APP_DIR`         | `/workspace`          | Path to application directory
-| `<buildpacks>`  | `CNB_BUILDPACKS_DIR`  | `/cnb/buildpacks`     | Path to buildpacks directory (see [Buildpacks Directory Layout](#buildpacks-directory-layout))
-| `<group>`       | `CNB_GROUP_PATH`      | `<layers>/group.toml` | Path to output group definition
-| `<layers>`      | `CNB_LAYERS_DIR`      | `/layers`             | Path to layers directory
-| `<log-level>`   | `CNB_LOG_LEVEL`       | `info`                | Log Level
-| `<order>`       | `CNB_ORDER_PATH`      | `/cnb/order.toml`     | Path to order definition (see [`order.toml`](#ordertoml-toml))
-| `<plan>`        | `CNB_PLAN_PATH`       | `<layers>/plan.toml`  | Path to output resolved build plan
-| `<platform>`    | `CNB_PLATFORM_DIR`    | `/platform`           | Path to platform directory
-
-##### Outputs
-| Output             | Description
-|--------------------|----------------------------------------------
-| [exit status]      | (see Exit Code table below for values)
-| `/dev/stdout`      | Logs (info)
-| `/dev/stderr`      | Logs (warnings, errors)
-| `<group>`          | Detected buildpack group  (see [`group.toml`](#grouptoml-toml))
-| `<plan>`           | Resolved Build Plan (see [`plan.toml`](#plantoml-toml))
-
-| Exit Code | Result|
-|-----------|-------|
-| `0`       | Success
-| `11`      | Platform API incompatibility error
-| `12`      | Buildpack API incompatibility error
-| `1-10`, `13-99` | Generic lifecycle errors
-| `100`     | All buildpacks groups have failed to detect w/o error
-| `101`     | All buildpack groups have failed to detect and at least one buildpack has errored
-| `102-199` | Detection-specific lifecycle errors
-
-The lifecycle:
-- SHALL detect a single group from `<order>` and write it to `<group>` using the [detection process](buildpack.md#phase-1-detection) outlined in the Buildpack Interface Specification
-- SHALL write the resolved build plan from the detected group to `<plan>`
-
 #### `analyzer`
 Usage: 
 ```
 /cnb/lifecycle/analyzer \
```

Comment:
The RFC mentions the addition of `-run-image`, destination tags, `stack`, and `cache-dir`. Should we be including these and the associated behavior (recording the run-image digest, validating write access etc.) in this PR? If not should we be creating separate issues for API 0.7?

Reply:
I wasn't _planning_ on adding those inputs in this update. I can create tickets. I could also be convinced we should do this now.

Reply:
I added the additional fields in https://github.com/buildpacks/spec/pull/197/commits/de9d5a694204abccfae24a746411e4939cf845c9.

Reply:
I re-removed these fields. Will followup with another spec PR to add them back and discuss what exactly we do with those fields.